docs/source/fr/pipeline_tutorial.md

```diff
@@ -0,0 +1,290 @@
+
+```py
+transcriber = pipeline(model="openai/whisper-large-v2", device=0, batch_size=2)
+audio_filenames = [f"https://huggingface.co/datasets/Narsil/asr_dummy/resolve/main/{i}.flac" for i in range(1, 5)]
+texts = transcriber(audio_filenames)
+```
+
+Cet exemple exécute le pipeline sur les 4 fichiers audio fournis, mais il les passera par lots de 2 au modèle (qui est sur un GPU, où le traitement par lots est plus susceptible d'aider) sans nécessiter de code supplémentaire de votre part.
+Le résultat sera le même que si vous aviez exécuté le pipeline sur chaque fichier audio individuellement. L'objectif est de vous aider à obtenir les résultats plus rapidement.
+
+Les pipelines peuvent également atténuer certaines des complexités du traitement par lots car, pour certains pipelines, un seul élément (comme un long fichier audio) doit être divisé en plusieurs parties pour être traité par un modèle. Le pipeline effectue ce [*traitement par lots de morceaux*](./main_classes/pipelines#pipeline-chunk-batching) pour vous.
+
+### Paramètres spécifiques à une tâche
+
+Toutes les tâches ont des paramètres spécifiques qui permettent une flexibilité et des options supplémentaires pour vous aider à compléter cette tâche.
+Par exemple, la méthode [`transformers.AutomaticSpeechRecognitionPipeline.__call__`] a un paramètre `return_timestamps` qui semble utile pour sous-titrer des vidéos :
+
+```py
+>>> transcriber = pipeline(model="openai/whisper-large-v2", return_timestamps=True)
+>>> transcriber("https://huggingface.co/datasets/Narsil/asr_dummy/resolve/main/mlk.flac")
+{'text': ' I have a dream that one day this nation will rise up and live out the true meaning of its creed.', 'chunks': [{'timestamp': (0.0, 11.88), 'text': ' I have a dream that one day this nation will rise up and live out the true meaning of its'}, {'timestamp': (11.88, 12.38), 'text': ' creed.'}]}
+```
+
+Comme vous pouvez le voir, le modèle a déduit le texte et a également indiqué **quand** les différentes phrases ont été prononcées.
+
+Il y a beaucoup de paramètres disponibles pour chaque tâche, donc consultez la documentation de l'API de chaque tâche pour voir ce que vous pouvez paramétrer !
+Par exemple, le pipeline [`~transformers.AutomaticSpeechRecognitionPipeline`] a un paramètre `chunk_length_s` qui est utile quand on travaille sur des fichiers audio très longs (par exemple, sous-titrage de films entiers ou de vidéos d'une heure) qu'un modèle ne peut généralement pas gérer seul :
+
+```python
+>>> transcriber = pipeline(model="openai/whisper-large-v2", chunk_length_s=30, return_timestamps=True)
+>>> transcriber("https://huggingface.co/datasets/sanchit-gandhi/librispeech_long/resolve/main/audio.wav")
+{'text': " Chapter 16. I might have told you of the beginning of this liaison in a few lines, but I wanted you to see every step by which we came.  I, too, agree to whatever Marguerite wished, Marguerite to be unable to live apart from me. It was the day after the evening...
```

Comment:
```suggestion
{'text': " Chapter 16. I might have told you of the beginning of this liaison in a few lines, but I wanted you to see every step by which we came.  I, too, agree to whatever Marguerite wished, Marguerite to be unable to live apart from me. It was the day after the evening...}
```

Reply:
@stevhliu  Good catch! It appears that the brace is also missing in the English version of the documentation. Should I create an issue for that?

Reply:
You can just add the missing brace for the English docs in this PR!